Dockerfile

```diff
@@ -1,16 +1,16 @@
-FROM golang:1.13.4-alpine as builder
-RUN apk add --update make
-ENV PATH /go/bin:/usr/local/go/bin:$PATH
-ENV GOPATH /go
-COPY . /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure
+FROM golang:1.13.10-alpine as builder
 WORKDIR /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure
-ARG IMAGE_VERSION=0.0.5
-RUN make build
+ADD . .
+ARG TARGETARCH
+ARG TARGETOS
+ARG LDFLAGS
+RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "${LDFLAGS}" -o _output/secrets-store-csi-driver-provider-azure ./cmd/
 
-FROM alpine:3.11.5
-RUN apk add --no-cache bash
+FROM us.gcr.io/k8s-artifacts-prod/build-image/debian-base-amd64:v2.1.0
 COPY --from=builder /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure/_output/secrets-store-csi-driver-provider-azure /bin/
 COPY --from=builder /go/src/github.com/Azure/secrets-store-csi-driver-provider-azure/install.sh /bin/install_azure_provider.sh
+RUN clean-install bash
```

Comment:
can we bring that scripts functionality inside the container or binary instead of adding a shell?